src/components/AddProfileForm.css

```diff
@@ -28,21 +22,17 @@
   height: 30px;
   width: 45px;
   border-radius: 10px;
```

Comment:
You set the border to 0 and also removed the transition. So I think the border-radius is no longer needed.
But it could be that you use a transition in another place and need the border-radius to be set here?